README.md

```diff
@@ -213,71 +284,24 @@ module "landing_zone" {
   }
 ```
 
-## Enable SNS topic subscription
+## SNS topic subscription
+
+| Topic Name  |  Variable | Content  |
+|---|---|---|
+| `aws-controltower-AggregateSecurityNotifications` | `sns_aws_config_subscription`  |  Aggregated AWS Config notifications |
+| `LandingZone-SecurityHubFindings` |  `sns_aws_security_hub_subscription` |  Aggregated Security Hub findings |
+| `LandingZone-IAMActivity` | `sns_monitor_iam_activity_subscription`  |  IAM activity findings |
 
-To subscribe to the `AggregatedSecurityNotifications` topic to receive security findings, set the `sns_security_subscription` variable as shown below.
 
 Example for https protocol and specified webhook endpoint:
 
 ```hcl
 module "landing_zone" {
   ...
   
-  sns_security_subscription = {
+  sns_aws_config_subscription = {
```

Comment:
I was thinking about this the other day, to propose grouping with other AWS variables by prefixing with `aws` (`aws_config_sns_subscription`).

This was in relation to variables such as:

* `aws_account_password_policy`
* `aws_config`
* `aws_create_account_password_policy`
* `aws_deny_leaving_org`
* `aws_deny_root_user_ous`
* `aws_ebs_encryption_by_default`
* `aws_guardduty`
* `aws_region_restrictions`
* `aws_require_imdsv2`
* `aws_required_tags`
* `aws_sso_permission_sets`

Reply:
I agree with you here, i'll change the code :-)